.github/actions/publish-wheel/action.yml

```diff
@@ -0,0 +1,51 @@
+name: 'Publish Wheel'
+description: 'TBD'
```

Comment:
You can remove if not required or add something here

Reply:
resolved in [e84827c](https://github.com/AuroTFP/aurotfp-github-actions/pull/1/commits/e84827c68d70269cea8e4cdb940be6276fdab14a)